components/Smiley.tsx

```diff
@@ -19,12 +19,25 @@ const EMOJI_MAP: EmojiMap = {
 }
 
 const Smiley = ({ percentage }: Props) => {
+  const emojiKeys = Object.keys(EMOJI_MAP).map(Number)
+  const [min, max] = [Math.min(...emojiKeys), Math.max(...emojiKeys)]
   const matchingEmoji = Object.keys(EMOJI_MAP).find(
     (key) => parseInt(key) >= percentage * 100
   )
   if (!matchingEmoji) return null
 
-  return <span className={styles.large}>{EMOJI_MAP[matchingEmoji]}</span>
+  return (
+    <span
+      aria-valuemax={max / 10}
+      aria-valuemin={min / 10}
+      aria-valuenow={parseInt(matchingEmoji) / 10}
+      className={styles.large}
+      role="slider"
```

Comment:
Is this a slider? The smiley sits above an actual slider that the user controls. Should the smiley maybe be aria-hidden entirely and the slider itself used to display percentage to the user?

Reply:
You're right, not sure why I added slider functionality to the emojis. Will clean this up